tests/integration/wallet/test_commands.py

```diff
@@ -10,9 +10,13 @@
 import lbryschema
 lbryschema.BLOCKCHAIN_NAME = 'lbrycrd_regtest'
 
+from torba.constants import COIN
+
+
```

Comment:
I don't think 2 blank lines is necessary between import statements. One is enough if you are trying to distinguish blocks of imports.

Reply:
Great!